src/model/MutableAreaDataSource.ts

```diff
@@ -376,7 +376,7 @@ export default class MutableAreaDataSource extends AreaDataSource {
    * @param newAreaName new area name
    * @param depth tree depth
    */
-  async updatePathTokens (session: ClientSession, changeRecord: ChangeRecordMetadataType, area: AreaDocumnent, newAreaName: string, depth: number = 2): Promise<void> {
+  async updatePathTokens (session: ClientSession, changeRecord: ChangeRecordMetadataType, area: AreaDocumnent, newAreaName: string, depth: number = 1): Promise<void> {
     if (area.pathTokens.length > 1) {
```

Comment:
I changed starting depth to 2 because we don't allow updating country name which has pathTokens.length === 1.  Is it causing an issue?

https://github.com/OpenBeta/openbeta-graphql/blob/2d79c70100a42a37c083ed575fd7d0d0dddbeb3d/src/model/MutableAreaDataSource.ts#L297

Reply:
The last checks showed some weird test results where it looked like we were using the wrong index. Since the first iteration updates the last item in the areas pathTokens, it should be [ pathTokens.length - 1 ] right? As it was, it was starting at [ length - 2 ]. I tested it locally before and after and was having issues with the 2.

Reply:
I just realized instead of passing depth down through the calls, we just have to know the index of the initial update and pass that index down to the children - in the case above, the index of Wasatch would be 2 in all the childrens pathTokens.

Reply:
I just realized that I don't need to index from the back (length - depth) because the result is constant. I just overthought this initially.

ex: Index of Wasatch is 2 in self and children
[USA, Utah, Wasatch]
[USA, Utah, Wasatch, Little Cottonwood]
[USA, Utah, Wasatch, Little Cottonwood, The Thumb]